akvo/rsr/models/user.py

```diff
@@ -376,16 +376,14 @@ def approved_employments(self):
         return self.employers.all().exclude(is_approved=False)
 
     def can_create_project(self):
-        """
-        Check to see if the user can create a project, or is a superuser.
+        """Check to see if the user can create a project."""
 
-        :return: Boolean to indicate whether the user has a reportable organisation
-        """
-        if self.is_superuser or self.is_admin:
+        if self.is_superuser:
```

Comment:
`self.has_perm` automatically gives all permissions to any user who is a `superuser`.  Also, if a user `is_admin` they have the `rsr.add_project` permissions given to them in our permission config.  So, I don't think this special case is required.